src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -1882,6 +1882,199 @@ public function get_test_plugin_theme_auto_updates() {
 		return $result;
 	}
 
+	/**
+	 * Test available disk space for updates.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @return array The test results.
+	 */
+	public function get_test_available_updates_disk_space() {
+		$available_space = function_exists( 'disk_free_space' ) ? @disk_free_space( WP_CONTENT_DIR . '/upgrade/' ) : false;
+
+		if ( false === $available_space ) {
+			$available_space = 0;
+		}
```

Comment:
`disk_free_space()` returns either a `float` or `false`; however, it's being used as an integer within the this method. For readability and PHP cross-version compatibility, the `float` should be explicitly type casted to `integer`.

```suggestion
		$available_space = false !== $available_space
			? (int) $available_space
			: 0;
```

More info on PHP cross-version compatibility:
- In PHP 8.1, implicitly coercing a `float` into `int` will throw a deprecation notice, whereas type casting to `integer` will not throw the notice. https://php.watch/versions/8.1/deprecate-implicit-conversion-incompatible-float-string
- The deprecation notice in PHP 8.x will become a fatal error in PHP 9.